### DISCUSSION TITLE
Hide content from pipes when stopped

### DISCUSSION THREAD

**Discussion by mauroalexandre** [frangoteam/FUXA#443] [Ideas]
First of all, thanks for your awesome project. I'm subscribed in patreon for contrib.

I'm making my first SCADA view using Fuxa and when I use pipes, I need hide de content from pipe when stopped to avoid user confusion.

It's possible? How?

Thanks

**Comment by unocelli** [comment]
Hi,
Thank you very much for supporting the FUXA project. I will take care of your request in the next few days.
